modules/terraform/Makefile

```diff
@@ -34,6 +34,17 @@ terraform/validate:
 terraform/tflint:
 	tflint --enable-plugin=aws
 
+## Auto-fix terraform deprecated list accessors
+terraform/tflint-fix-deprecated-index:
+	@for v in $$(find . -type f -name '*.tf'); do \
+		sed -i 's/\([[:alpha:]][-_[:alpha:][:digit:]]\{2,\}\)\.\([*0-9]\+\)\./\1[\2]\./g' "$$v" ; \
```

Comment:
Sadly, this doesn't work on macOS. I think on macOS, character classes like `[:alpha:]` are not recognized in normal mode, and `-i` requires an argument. Not quite sure what to do about it.

Reply:
let's call only inside precommit/terraform which is inside the container?